cumulus_library_suicidality_icd10/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.1.0"
```

Comment:
Minor bump is fine, but in my head a major bump on the runner might be worth a major bump here too? Don't actually care, just noting it.

Reply:
fair point, i'll update